gremlin-test/features/map/MergeEdge.feature

```diff
@@ -0,0 +1,201 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+@StepClassMap @StepMergeE
+Feature: Step - mergeE()
```

Comment:
Please add the following test scenarios (similar cases for mergeV as well):
1. Uses g.mergeE() but does not specify OUT in the map
2. Uses g.mergeE() but does not specify IN in the map 
3. Uses g.mergeE() but does not specify either OUT or IN in the map 
4. Uses g.mergeE() but does not specify LABEL in the map 
5. Uses g.V(1).mergeE() but specifies OUT in the map which is different than 1
6. Wrong syntax for the map (this case could go in grammar test)
7. lower case OUT/IN in the map (this case could go in grammar test) 
8. test for mergeE(traversal) --> (I am quite curious to see the use case for this scenario)
9. mergeE() at the beginning of a child traversal
10. mergeE() in the middle a child traversal 
11. mergeV() with insertion of property having different cardinality (list/set/single)
12. mergeV() when existing vertex has single cardinality but new vertex wants to add same property with set cardinality

Reply:
thanks for all the test ideas. this was an area i needed to return to. adding a test index as comments to each test file to make it easier to understand what is being tested as well